#include <QMainWindow>
#include <iostream>
#include <QTableWidgetItem>
#include <QShortcut>
#include <QMessageBox>


QT_BEGIN_NAMESPACE
namespace Ui { class MainWindow; }
QT_END_NAMESPACE

class MainWindow : public QMainWindow
{
    Q_OBJECT

public:
    MainWindow(QWidget *parent = nullptr);
    ~MainWindow();

private slots:
    void on_R_clicked();

    void on_Q_clicked();

    void on_tableWidget_itemChanged(QTableWidgetItem *item);

private:
    Ui::MainWindow *ui;
    QShortcut *R;
    QShortcut *Q;
};
